### ISSUE TITLE
Include directories not added to consuming project

### ISSUE THREAD

**Issue by willyd** [coapp/coapp.powershell#45]
Include directories are not automatically included in the consuming project with coapp powershell v2.4.4.425 as it was in the release version. 

[Edit]

I found that this bug happens only when we use the nested include feature. If I create a package with 

```
nestedInclude: {
        #destination = ${d_include}\mylib;
        "root\mylib\*"
    };
```

then no `AdditionalIncludeDirectories` is added to the targets file.

```
    nestedInclude: {
        #destination = ${d_include}\mylib;
        "root\mylib\*"
    };

    include: { "root\*" };
```

and there is at least one file in root the `AdditionalIncludeDirectories` is properly added to the targets file.

**Comment by fearthecowboy** [comment]
Does this only happen if there is no content in the include folder itself?

**Comment by willyd** [comment]
Yes. If I include a dummy header in the include folder then the `AdditionalIncludeDirectories` is added as expected.

**Comment by fearthecowboy** [comment]
Ok, then I think I know what's going on. Lemme see if I can fix this.

**Comment by fearthecowboy** [comment]
Hmm. Oddly enough, this isn't a simple fix. 

The trouble is, in the generated package data, I haven't a correlation between the `include` directory and any synthetically-added `nestedInclude` folders.

So, when there isn't any content in the include folder itself, it doesn't add it to the include path.

I think you're going to have to add a line to the `targets` section 

```
targets {
    Includes += ${pkg_root}/${d_include}; 
};

```

**Comment by willyd** [comment]
Ok. I'll do that. Still better than adding a dummy header file.
